commcare_connect/opportunity/api/serializers.py

```diff
@@ -105,6 +114,15 @@ def get_deliver_progress(self, obj):
         opp_access = _get_opp_access(self.context.get("request").user, obj)
         return opp_access.visit_count
 
+    def get_max_visits_per_user(self, obj):
+        return obj.paymentunit_set.aggregate(max_total=Sum("max_total")).get("max_total", 0)
+
+    def get_daily_max_visits_per_user(self, obj):
+        return obj.paymentunit_set.aggregate(max_daily=Sum("max_daily")).get("max_daily", 0)
```

Comment:
I think its safest to use `max(max_daily)` since payment units often represent sequential visits. This is all temporary until we update the mobile anyway, but it will be slightly more accurate.

Reply:
Consider below
- PU1 (max_daily=6)
- PU2 (max_daily=5)

You are saying we should return 6 max_daily for now? I would expect 11 visits to be allowed per day. I guess that's fine temporarily (ideally we should send these as per payment units and mobile should handle the same)

Reply:
Yeah, I agree that the real answer is definitely on mobile. In the meantime, it sounds like Rama agrees with you, so I hate to say this, but can you change it back before merging?

Reply:
Cool. Is it just here or the other place as well? https://github.com/dimagi/commcare-connect/pull/298#discussion_r1561887527